config/locales/errors/es.yml

```diff
@@ -53,8 +54,8 @@ es:
       general_error: No pudimos leer ni verificar su identificación. Intenta tomar
         y subir nuevas imágenes de la parte delantera y trasera de tu ID. Asegúrese
         de que todo se pueda leer fácilmente, incluido el código de barras.
-      invalid_file_input_type: No se acepta este tipo de archivo, elija un archivo
-        JPG, PNG, BMP o TIFF.
+      invalid_file_input_type: Ese formato no es admitido, por favor elija un archivo
+        JPG, PNG, BMP, o TIFF.
```

Comment:
Something possibly worth following-up on: We're not always consistent with our use of Oxford comma. There's very similar error messages where we omit it (e.g. "Must be a JPG, BMP, HEIC, PNG or TIFF", translated as "Debe ser un JPG, BMP, HEIC, PNG o TIFF").

Since this is correct per our own English text and translation, I don't think it's a blocker.